### ISSUE TITLE
can I name SSRC something other than a random hexadecimal?

### ISSUE THREAD

**Issue by alatteri** [CESNET/UltraGrid#417]
Is there a way to name SSRC XXXXXXXX something other than a random hexadecimal (SSRC 0x61997a73)??
The reason I ask, is I am parsing the log file, and since the SSRC name is random, it is hard to parse the video and audio streams separately.

Thanks.

**Comment by alatteri** [comment]
Hi Martin... yeah that would be fine.  Anyway, that is consistent to delineate between the audio and video stream.

Thanks.

**Comment by MartinPulec** [comment]
Good, in this case I believe I've already done that.
